ArrOperation.java

```diff
@@ -0,0 +1,260 @@
+package Assignment4;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This Class implements methods to perform operation on positive integer array
+ * @author Sabir
+ *
+ */
+public class ArrOperation {
+
+	/**
+	 * Find size of largest mirror section in positive integer array
+	 * @param inputArray an array of positive integer
+	 * @return size of largest mirror section in the array
+	 * @throws AssertionError when array is empty 
+	 * @throws AssertionError when array contain negative number

```

Comment:
Use single "@throws AssertionError" and add both condition detail. Use different throw statement if different type of exception.

Reply:
ok sir